app/api/helpers/tasks.py

```diff
@@ -120,6 +121,20 @@ def resize_user_images_task(self, user_id, original_image_url):
         logging.exception('Error encountered while generating resized images for user with id: {}'.format(user_id))
 
 
+@celery.task(base=RequestContextTask, name='sponsor.logo.urls', bind=True)
+def sponsor_logos_url_task(self, event_id):
+    sponsors = Sponsor.query.filter_by(event_id=event_id).all()
```

Comment:
is_active=True

Reply:
@iamareebjamal Didn't get it :sweat_smile:

Reply:
@iamareebjamal  Did you mean it like this?

Reply:
You should only get sponsors where is_active is True, getting all means you'll get deleted sponsors as well